__tests__/app.test.js

```diff
@@ -118,4 +119,55 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe('Route not found')
       })
   })
+})
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for a given article_id, each comment with correct properties", () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toHaveLength(11)
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject(
+            {
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            }
+          )
+        })
+      })
+  })
+
+  test("404: if no comments are found for the given article_id, respond with an errror", () => {
+    return request(app)
+      .get('/api/articles/2/comments')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('No comments')
+      })
+  })
```

Comment:
The above test should return a 200 and an empty array - because the article does exist it just doesn't have any comments attached to it yet. You will need to check that the article does exist in this case!